### ISSUE TITLE
Numeral Styles

### ISSUE THREAD

**Issue by davelab6** [MichalSahar/Miriam-Libre#2]
There are regular numerals are lining style and proportionally spaced. 

In the future it would be great to have all 4 styles:
- [x] lining proportional
- [ ] lining tabular
- [ ] old style proportional
- [ ] old style tabular

Also the font today has nice latin numerals made for the Hebrew height. These are in the small caps opentype code (with the `.sc` in the glyph names). It would be great to add all 4 numeral styles for these as well...
- [ ] lining proportional
- [ ] lining tabular
- [ ] old style proportional
- [ ] old style tabular

But some care needs to be taken with the opentype code. 
- [ ] Also, add opentype `locl` code so these are used when the language selection is used (which doesn't work in all common software yet)

There is a glyphs tutorial on this at https://glyphsapp.com/tutorials/figure-sets

**Comment by davelab6** [comment]
To add tabular numberals I do the following

In each master,
- copied the numerals
- selected them and use find and replace to replace `.001` with `.tf`
- look for the widest number in the family's width, the zero in the bold is `625`
- enter each .tf glyph and clear its contents, and add a component
- right click the component and Disable Automatic Alignment

<img width="667" alt="screen shot 2016-03-16 at 12 00 25" src="https://cloud.githubusercontent.com/assets/261579/13819224/291b3c2e-eb6f-11e5-8ed3-c27dbb2aa55e.png">
- set the width to 625, and then move the component to be centered
- right click the glyph and run `Autohint`

Then go to Font Info, Features, and click the 'Update' button at the bottom of the window, to add the new features

I'll make a pull request shortly

**Comment by mushon** [comment]
Hey @davelab6 I actually need this for a site I'm working on which is heavy on budgetary data (obudget.org). I encourage you to issue that pull request and would you please expedite the inclusion of the updated version into fonts.Google.com?

**Comment by davelab6** [comment]
I did https://github.com/MichalSahar/Miriam-Libre/pull/3

The rest had to be done by Michal or someone else. Can obudget.org fund this work? It's not much

**Comment by mushon** [comment]
oBudget.org is a volunteer led project with a very limited budget. We're not really in a position to give corporate sponsorship to open source projects, as we're an open source project ourselves. @davelab6 can you specify exactly what's required at this point in order for us (and maybe others) to be able to use the Tabular Figures?

**Comment by davelab6** [comment]
Ideally Michal or someone needs to make the full set of numbers, then
Michal needs to cut a release, and I need to push it :)

**Comment by mushon** [comment]
What do you mean by "the full set of numbers"? Are the Tabular Lining numbers currently available, or do they require more work? When you say "full set" do you mean other than just 0-9 tabular lining?

**Comment by davelab6** [comment]
There are 4 sets of numbers possible, and now we have the original and the
tabular ones I contributed. The other 2 are "old style"

However if you don't need them, we can go from here :)

@m4rc1e could you queue this up?

**Comment by MichalSahar** [comment]
I will take care of it. No problem.

**Comment by m4rc1e** [comment]
@MichalSahar could you also merge my pr before you do this?

#4

**Comment by mushon** [comment]
Yes, I only need the tabular lining. Old Style is not holding me back and frankly I'm not sure it is necessarily needed for Miriam Libre. I might be wrong though… I wonder, is there a best practice requiring Old Style figures for every font whatsoever?

**Comment by robsonsobral** [comment]
Any chance of merging and releasing, @MichalSahar ? Thank you!
